compiler/lib/code.ml

```diff
@@ -697,6 +730,23 @@ let fold_children blocks pc f accu =
       let accu = Array.fold_right ~init:accu ~f:(fun (pc, _) accu -> f pc accu) a1 in
       accu
 
+let fold_children_skip_try_body blocks pc f accu =
```

Comment:
This function seems to exist in Tailcall and Inline as `fold_children`. We should remove this duplication.

Reply:
Oh, right! That makes even more sense to add it to `Code`. then.

Reply:
Done